### PR TITLE
Fix PHPCS

### PR DESCRIPTION
**Pull request by abhijitrakas**
Fix PHPCS errors. Some errors are still there which needs more attention and testing so not fixing now.

### CODE REVIEW

**Comment by abhijitrakas** [comment]
@chandrapatel Please review this PR once you are free.

File: admin/js/nginx-helper-admin.js [hunk 1]
```diff
@@ -1,92 +1,110 @@
...
+
+				var args = {
+					'action': 'rt_get_feeds'
+				};
+
+				jQuery.get(
+					ajaxurl,
+					args,
+					function (data) {
+						news_section.find( '.inside' ).html( data );
```
Comment (thrijith) [inline admin/js/nginx-helper-admin.js]: Avoid use of `.html(` here.

Comment (thrijith) [reply admin/js/nginx-helper-admin.js]: Remove `console.log` as well.

File: admin/js/nginx-helper-admin.js [hunk 2]
```diff
@@ -1,92 +1,110 @@
...
+						console.log( data );
+					}
+				);
+
+			}
+
+			jQuery( "form#purgeall a" ).click(
+				function (e) {
+
+					if ( confirm( "Purging entire cache is not recommended. Would you like to continue ?" ) == true ) {
```
Comment (thrijith) [inline admin/js/nginx-helper-admin.js]: This string should be translatable.

File: admin/js/nginx-helper-admin.js [hunk 3]
```diff
@@ -1,92 +1,110 @@
...
+
+				jQuery( '#' + selector ).on(
+					'change',
+					function () {
+
+						if ( jQuery( this ).is( ':checked' ) ) {
+
+							jQuery( '.' + selector ).show();
+
+							if ( selector == "cache_method_redis" ) {
```
Comment (thrijith) [inline admin/js/nginx-helper-admin.js]: Could we use string checks here? Use single quote for literal strings.

File: admin/partials/nginx-helper-admin-display.php [hunk 4]
```diff
@@ -35,28 +35,35 @@
 					echo wp_kses(
 						sprintf(
 							'<a class="nav-tab%1$s" href="?page=nginx&tab=%2$s">%3$s</a>',
-							esc_attr( $class ), esc_attr( $setting_name['menu_slug'] ), esc_html( $setting_name['menu_title'] )
```
Comment (thrijith) [inline admin/partials/nginx-helper-admin-display.php]: Same as previous use case of `sprintf`. 

Could we concatenate `$setting_name['menu_slug'] ` with static value in `href` and pass it to `esc_url` instead?

File: admin/partials/nginx-helper-general-options.php [hunk 5]
```diff
@@ -118,12 +118,21 @@
 							<label for="cache_method_fastcgi">
 								<?php
 								echo wp_kses(
-										sprintf(
-											'%1$s (<a target="_blank" href="%2$s" title="%3$s">%4$s</a>)',
-											esc_html__( 'nginx Fastcgi cache', 'nginx-helper' ), esc_url( $nginx_setting_link ), esc_attr__( 'External settings for nginx', 'nginx-helper' ), esc_html__( 'requires external settings for nginx', 'nginx-helper' )
+									sprintf(
```
Comment (thrijith) [inline admin/partials/nginx-helper-general-options.php]: Same as previous use case of `sprintf`.

File: admin/partials/nginx-helper-sidebar-display.php [hunk 6]
```diff
@@ -36,7 +36,9 @@
 			echo wp_kses(
 				sprintf(
 					'%1$s <a href=\'%2$s\'>%3$s</a>.',
-					esc_html__( 'Please use our', 'nginx-helper' ), esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ), esc_html__( 'free support forum', 'nginx-helper' )
+					esc_html__( 'Please use our', 'nginx-helper' ),
+					esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ),
```
Comment (thrijith) [inline admin/partials/nginx-helper-sidebar-display.php]: escaping not needed here.

**Comment by abhijitrakas** [comment]
@thrijith Updated code as per suggested please check.

File: admin/js/nginx-helper-admin.js [hunk 7]
```diff
@@ -1,92 +1,110 @@
...
+
+				jQuery( '#' + selector ).on(
+					'change',
+					function () {
+
+						if ( jQuery( this ).is( ':checked' ) ) {
+
+							jQuery( '.' + selector ).show();
+
+							if ( selector === 'cache_method_redis' ) {
```
Comment (thrijith) [inline admin/js/nginx-helper-admin.js]: @abhijitrakas minor change, might need a Yoda check here rest LGTM.

**Comment by thrijith** [comment]
Conflicts have been resolved and remaining CS issue has been fixed. @chandrapatel please review and merge, if the changes look ok to you.
Thanks

File: admin/partials/nginx-helper-general-options.php [hunk 8]
```diff
@@ -287,9 +293,14 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>%5$s<strong>%6$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'modified', 'nginx-helper' ), esc_html__( ' or ', 'nginx-helper' ), esc_html__( 'added', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'modified', 'nginx-helper' ),
+												esc_html__( ' or ', 'nginx-helper' ),
+												esc_html__( 'added', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 9]
```diff
@@ -311,9 +322,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'published post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'trashed', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'published post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'trashed', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 10]
```diff
@@ -345,9 +359,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'post', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'published', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'post', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'published', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 11]
```diff
@@ -369,9 +386,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'comment', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'approved/published', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'comment', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'approved/published', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 12]
```diff
@@ -393,9 +413,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'comment', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'unapproved/deleted', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'comment', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'unapproved/deleted', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 13]
```diff
@@ -429,9 +452,14 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>%5$s<strong>%6$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'modified', 'nginx-helper' ), esc_html__( ' or ', 'nginx-helper' ), esc_html__( 'added', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'modified', 'nginx-helper' ),
+												esc_html__( ' or ', 'nginx-helper' ),
+												esc_html__( 'added', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 14]
```diff
@@ -453,9 +481,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'published post', 'nginx-helper' ), esc_html__( ' (or page/custom post) is ', 'nginx-helper' ), esc_html__( 'trashed', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'published post', 'nginx-helper' ),
+												esc_html__( ' (or page/custom post) is ', 'nginx-helper' ),
+												esc_html__( 'trashed', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 15]
```diff
@@ -478,9 +509,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'comment', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'approved/published', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'comment', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'approved/published', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 16]
```diff
@@ -502,9 +536,12 @@
 										echo wp_kses(
 											sprintf(
 												'%1$s<strong>%2$s</strong>%3$s<strong>%4$s</strong>.',
-												esc_html__( 'when a ', 'nginx-helper' ), esc_html__( 'comment', 'nginx-helper' ), esc_html__( ' is ', 'nginx-helper' ), esc_html__( 'unapproved/deleted', 'nginx-helper' )
+												esc_html__( 'when a ', 'nginx-helper' ),
+												esc_html__( 'comment', 'nginx-helper' ),
+												esc_html__( ' is ', 'nginx-helper' ),
+												esc_html__( 'unapproved/deleted', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 17]
```diff
@@ -604,9 +646,13 @@
 						echo wp_kses(
 							sprintf(
 								'%1$s<br /><small>%2$s</small>',
-								esc_html__( 'Nginx Map path to include in nginx settings', 'nginx-helper' ), esc_html__( '(recommended)', 'nginx-helper' )
+								esc_html__( 'Nginx Map path to include in nginx settings', 'nginx-helper' ),
+								esc_html__( '(recommended)', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 18]
```diff
@@ -620,9 +666,14 @@
 							echo wp_kses(
 								sprintf(
 									'%1$s<br />%2$s<br /><small>%3$s</small>',
-									esc_html__( 'Or,', 'nginx-helper' ), esc_html__( 'Text to manually copy and paste in nginx settings', 'nginx-helper' ), esc_html__( '(if your network is small and new sites are not added frequently)', 'nginx-helper' )
+									esc_html__( 'Or,', 'nginx-helper' ),
+									esc_html__( 'Text to manually copy and paste in nginx settings', 'nginx-helper' ),
+									esc_html__( '(if your network is small and new sites are not added frequently)', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-general-options.php [hunk 19]
```diff
@@ -659,9 +710,14 @@
 					echo wp_kses(
 						sprintf(
 							'%1$s<br /><br />%2$s<strong>%3$s</strong>',
-							esc_html__( 'Can\'t write on log file.', 'nginx-helper' ), esc_html__( 'Check you have write permission on ', 'nginx-helper' ), esc_url( $log_path . 'nginx.log' )
+							esc_html__( 'Can\'t write on log file.', 'nginx-helper' ),
+							esc_html__( 'Check you have write permission on ', 'nginx-helper' ),
+							esc_url( $log_path . 'nginx.log' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-general-options.php]: This is not correct way to make small string translatable. We need to allow entire string translatable.

File: admin/partials/nginx-helper-sidebar-display.php [hunk 20]
```diff
@@ -33,12 +33,11 @@
...
-				sprintf(
-					'%1$s <a href=\'%2$s\'>%3$s</a>.',
-					esc_html__( 'Please use our', 'nginx-helper' ), esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ), esc_html__( 'free support forum', 'nginx-helper' )
-				),
-				array( 'a' => array( 'href' => array() ) )
+			printf(
+				'%s <a href=\'%s\'>%s</a>.',
+				esc_html__( 'Please use our', 'nginx-helper' ),
+				esc_url( 'http://rtcamp.com/support/forum/wordpress-nginx/' ),
+				esc_html__( 'free support forum', 'nginx-helper' )
```
Comment (chandrapatel) [inline admin/partials/nginx-helper-sidebar-display.php]: xThis is not correct way to make small string translatable. We need to allow entire string translatable.

**Comment by chandrapatel** [comment]
> Conflicts have been resolved and remaining CS issue has been fixed. @chandrapatel please review and merge, if the changes look ok to you.
> Thanks

@thrijith I have left feedback for translatable strings but we will fix it new PR and I'm merging this PR.

**Pull request merged**